File: examples/ruby/constructs/for_loop_control.rb
# Ported from examples/lumen/constructs/for_loop_control.lm by scripts/port_examples.py; edit the Lumen original, not this file.
for i in 0...15 do
    if i == 10 then
        break
    end
    print(i)
    if i < 9 then
        print(", ")
    end
end
puts("")

File: examples/c/constructs/for_loop_control.c
// Ported from examples/lumen/constructs/for_loop_control.lm by scripts/port_examples.py; edit the Lumen original, not this file.
int main(void) {
    long i = 0;
    while (i < 15) {
        if (i == 10) {
            break;
        }
        printf("%ld", i);
        if (i < 9) {
            printf(", ");
        }
        i = i + 1;
    }
    puts("");
    return 0;
}
